themes/docsy/layouts/practices/single.html

```diff
@@ -0,0 +1,121 @@
+{{ define "main" }}
+{{ partial "amplitude.html" (dict "context" . "event" "sample viewed") }}
```

Comment:
Let's remove this for now, or change the event type to "practice viewed" or something. This is metrics tracking for the Amplitude tool.

Reply:
Removed amplitude but kept the main wrapper on line 1 and end of file.